addons/test_discuss_full/tests/test_performance.py

```diff
@@ -459,6 +459,7 @@ def _get_init_messaging_result(self):
             "discuss.channel.rtc.session": [
                 self._expected_result_for_rtc_session(self.channel_channel_group_1, self.users[2]),
             ],
+            "res.groups": [{'full_name': 'Role / Member', 'id': self.channel_channel_group_1.group_public_id.id}],
```

Comment:
Group should not be read from channel (this makes the test cheating), but based on what we except in regards to the test setup (so `self.env.ref("base.group_user").id`).